molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/copyright/element.html

```diff
@@ -0,0 +1,9 @@
+{% raw %}
+{% load mote_tags %}
+
+<div class="footer">
+    <p class="footer__copyright">
+      © 2016 UNFPA, Ford Foundation. All Rights Reserved.
```

Comment:
Can we use a generic text here, maybe just `© All Rights Reserved.` or something like that